app/components/layout/top-nav.tsx

```diff
@@ -1,240 +1,105 @@
-                  >
-                    <span className="sr-only">Open user menu</span>
-                    <img
-                      className="h-8 w-8 rounded-full"
-                      src="https://images.unsplash.com/photo-1472099645785-5658abf4ff4e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80"
-                      alt=""
-                    />
-                  </button>
-                </div>
-
-                {/* <!--
-              Dropdown menu, show/hide based on menu state.
-
-              Entering: "transition ease-out duration-100"
-                From: "transform opacity-0 scale-95"
-                To: "transform opacity-100 scale-100"
-              Leaving: "transition ease-in duration-75"
-                From: "transform opacity-100 scale-100"
-                To: "transform opacity-0 scale-95"
-            --> */}
-                <div
-                  className="absolute right-0 z-10 mt-2 w-48 origin-top-right rounded-md bg-white py-1 shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none"
-                  role="menu"
-                  aria-orientation="vertical"
-                  aria-labelledby="user-menu-button"
-                  tabIndex="-1"
-                >
-                  {/* <!-- Active: "bg-gray-100", Not Active: "" --> */}
-                  <a
-                    href="/"
-                    className="block px-4 py-2 text-sm text-gray-700"
-                    role="menuitem"
-                    tabIndex="-1"
-                    id="user-menu-item-0"
-                  >
-                    Your Profile
-                  </a>
-                  <a
-                    href="/"
-                    className="block px-4 py-2 text-sm text-gray-700"
-                    role="menuitem"
-                    tabIndex="-1"
-                    id="user-menu-item-1"
-                  >
-                    Settings
-                  </a>
-                  <a
-                    href="/"
-                    className="block px-4 py-2 text-sm text-gray-700"
-                    role="menuitem"
-                    tabIndex="-1"
-                    id="user-menu-item-2"
-                  >
-                    Sign out
-                  </a>
-                </div>
-              </div>
-            </div>
+    <nav className="bg-gray-800 p-4">
+      <div className="flex justify-start items-center max-w-screen-2xl mx-auto">
+        <div className="text-white text-xl font-semibold flex justify-between items-center py-4 w-full">
+          <a href="/"><img  className="w-1/3" src="https://res.cloudinary.com/dxctpvd8v/image/upload/v1683267003/SocialPlanIt/SocialPlan-it-logo-Horizontal_xwm3xt.png" alt="Social Plan It horizontal logo" /></a>
+          <div className="hidden md:flex justify-between w-3/4 py-4" >
+          <ul className='flex'>
+            <a href="/"><li className='text-2xl py-1 hover:py-0'>Home</li></a>
```

Comment:
Let's use the Remix component "Link" to replace our <a> tags to get the client-side rendering benefits of Remix 💪